Project.toml

```diff
@@ -10,7 +10,7 @@ Requires = "ae029012-a4dd-5104-9daa-d747884805df"
 Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
 
 [compat]
-Requires = "0.5"
```

Comment:
Can we allow both?

Reply:
I guess not. AFAICT, there's just some performance benefits in 1.0.